src/index.ts

```diff
@@ -33,7 +33,7 @@ function genericCombinator(tpe: Tpe): Reader<Ctx, gen.CustomCombinator> {
   return ask<Ctx>().chain(({ prefix }) => {
     const type = gen.identifier(`${prefix}${tpe.name}`);
     const staticArgsR = traverseReader(tpe.args!, t => getType(t)).map(typeReferences =>
-      typeReferences.map(gen.printStatic).join(', ')
+      typeReferences.map(gen.printStatic)
     );
     const runtimeArgsR = traverseReader(tpe.args!, t => getType(t)).map(typeReferences =>
       typeReferences.map(gen.printRuntime).join(', ')
```

Comment:
not the best in terms of readability, since `runtimeArgsR` is instead a string (already `join`ed)